tsconfig.json

```diff
@@ -44,7 +44,7 @@
     "declarationMap": true,
     "strictNullChecks": true,
     "suppressImplicitAnyIndexErrors": true,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
Not sure if this is the real solution as we have other components with onClick methods in stories...